tests/test_unauthenticated_reddit.py

```diff
@@ -103,6 +106,48 @@ def test_get_front_page(self):
         num = 50
         self.assertEqual(num, len(list(self.r.get_front_page(limit=num))))
 
+    def test_load_config(self):
+        from praw import settings
```

Comment:
When it was outside the repeated reloading seemed to affect other tests when I first wrote this. I'll check if this is still the case.